circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -175,7 +175,7 @@ enum GeneratorIndex {
      * Indices with size ≤ 44
      */
     VK = 41,                        // Size = 35
-    PRIVATE_CIRCUIT_PUBLIC_INPUTS,  // Size = 39
+    PRIVATE_CIRCUIT_PUBLIC_INPUTS,  // Size = 45
```

Comment:
Jean changed this.... Not clear to me why :thinking:

Reply:
it actually should have been 41, then add 4 for addition of nullified commitments